cli-tests/src/upload.rs

```diff
@@ -33,7 +33,6 @@ async fn upload_bundle() {
         .env("GITHUB_JOB", "test-job")
         .args(&[
             "upload",
-            "--use-quarantining",
```

Comment:
No longer need to specify since this is set by default

Reply:
We will still need a way to skip the quarantining step.

Reply:
One may pass `--use-quarantining false` :)